### PR TITLE
Add endpoint for get_smoking_tracker

### PR DESCRIPTION
**Pull request by svenvanderburg**

### CODE REVIEW

File: niceday_client/niceday_client.py [hunk 1]
```diff
@@ -160,3 +161,25 @@ def set_user_tracker_statuses(self, user_id: int, tracker_statuses: typing.List[
...
+
+        Returns:
+            A list of smoking tracker entries, each entry is a dict containing
+            amongst others the startTime & endTime, value (which is itself a dict
+            containing the key 'quantity' that depicts the number of cigarettes smoked
+            in an entry).
+
+        """
+        url = self._niceday_api_uri + 'usertrackers/smoking/' + str(user_id)
+        query_params = {'startTime': start_time.isoformat() + 'Z',
```
Comment (raar1) [inline niceday_client/niceday_client.py:183]: Interesting that you need to add the Z manually

File: tests/test_niceday_client.py [hunk 2]
```diff
@@ -57,3 +58,15 @@ def test_set_user_tracker_statuses():
     client.set_user_tracker_statuses(
         user_id=existing_user_id,
         tracker_statuses=[TrackerStatus(trackerId=Tracker.smoking, isEnabled=True)])
+
+
+@pytest.mark.integration
+def test_get_smoking_tracker():
+    client = NicedayClient()
+    existing_user_id = 38527  # Please change this to your own test user id if used frequently
+    client.get_smoking_tracker(
```
Comment (raar1) [inline tests/test_niceday_client.py]: Should the return of this not be checked too?

**Review by raar1 — APPROVED** [review]
OK makes sense regarding PR#3. Just left a comment about the test :)

File: niceday_client/niceday_client.py (hunk 1 above)
Comment (svenvanderburg) [reply niceday_client/niceday_client.py:183]: Yes, that's weird right. But the internet gives no sensible other solutions.

File: tests/test_niceday_client.py (hunk 2 above)
Comment (svenvanderburg) [reply tests/test_niceday_client.py]: Yeah, good point. I added some asserts, but they are of course dependent on the niceday database.

**Pull request merged**